config/src/test/resources/CustomJdbcUserServiceSampleConfig.sql

```diff
@@ -1,4 +1,4 @@
-create table users(principal varchar_ignorecase(50) not null primary key,credentials varchar_ignorecase(50) not null);
+create table users(principal varchar_ignorecase(50) not null primary key,credentials varchar_ignorecase(100) not null);
```

Comment:
Would it be ok to rewrite this to ANSI  sql? varchar_ignorecase is ... not?

Reply:
You are welcome to rewrite in a separate commit. Code cleanups are always welcome so long as they are separate from the commit that adds XYZ feature.